docs/components/authentication/sign-in.mdx

```diff
@@ -23,54 +18,80 @@ The `<SignIn />` component renders a UI for signing in users. The functionality
 
 Below is basic implementation of the `<SignIn />` component. You can use this as a starting point for your own implementation.
 
-<CodeBlockTabs options={['Next.js', 'React', 'Remix', 'Gatsby', 'JavaScript']}>
-```jsx copy filename="/pages/sign-in/[[...index]].tsx"
-import { SignIn } from "@clerk/nextjs";
-
-const SignInPage = () => (
-  <SignIn path="/sign-in" routing="path" signUpUrl="/sign-up" />
-);
-export default SignInPage;
-```
-```jsx copy filename="/pages/sign-in/[[...index]].tsx"
-import { SignIn } from "@clerk/react";
-
-const SignInPage = () => (
-  <SignIn path="/sign-in" routing="path" signUpUrl="/sign-up" />
-);
-export default SignInPage;
-```
-```jsx copy filename="app/routes/sign-in/$.tsx"
-import { SignIn } from "@clerk/remix";
-
-export default function SignInPage() {
-  return (
-    <div style={{ border: "2px solid blue", padding: "2rem" }}>
-      <h1>Sign In route</h1>
-      <SignIn routing={"path"} path={"/sign-in"} />
-    </div>
-  );
-}
-```
-```jsx copy filename="/pages/sign-in.js"
-import { SignIn } from "gatsby-plugin-clerk";
-
-export default function SignInPage() {
-  return (
-    <div style={{ border: "2px solid blue", padding: "2rem" }}>
-      <h1>Sign In Up route</h1>
-      <SignIn routing={"path"} path={"/sign-in"} />
-    </div>
-  );
-}
-```
-```js copy filename="sign-in.js"
-window.Clerk.mountSignIn(
-  document.getElementById("sign-in")
-);
-window.Clerk.openSignIn();
-````
-</CodeBlockTabs>
+<Tabs items={['Next.js', 'React', 'Remix', 'Gatsby', 'JavaScript']}>
+  <Tab>
+    You can embed the `<SignIn />` component using the [Next.js optional catch all route](https://nextjs.org/docs/pages/building-your-application/routing/dynamic-routes#optional-catch-all-routes). This allows you to redirect the user inside your application. The `<SignIn />` component should be mounted on a public page.
+
+    <CodeBlockTabs options={["Pages router", "App router"]}>
+      ```jsx copy filename="/pages/sign-in/[[...index]].tsx"
+      import { SignIn } from "@clerk/nextjs";
+
+      const SignInPage = () => (
+        <SignIn path="/sign-in" routing="path" signUpUrl="/sign-up" />
+      );
+      export default SignInPage;
+      ```
+
+      ```jsx copy filename="/app/sign-in/[[...sign-in]]/page.tsx"
+      import { SignIn } from "@clerk/nextjs";
+
+      export default function Page() {
+        return <SignIn />;
+      }
+      ```
+    </CodeBlockTabs>
```

Comment:
Let's lead with the App Router 👀

```suggestion
    <CodeBlockTabs options={["App Router", "Pages Router"]}>
      ```jsx copy filename="/app/sign-in/[[...sign-in]]/page.tsx"
      import { SignIn } from "@clerk/nextjs";

      export default function Page() {
        return <SignIn />;
      }
      ```

      ```jsx copy filename="/pages/sign-in/[[...index]].tsx"
      import { SignIn } from "@clerk/nextjs";

      const SignInPage = () => (
        <SignIn path="/sign-in" routing="path" signUpUrl="/sign-up" />
      );
      export default SignInPage;
      ```
    </CodeBlockTabs>
```